server/server.js

```diff
@@ -116,8 +116,12 @@ app.post(`${apiBaseUrl}/employees`,
       return res.json({ status: 'Error', message: req.fileError });
     }
 
-    const response = await parseAndSaveParticipants(req.file);
-    return res.json(response);
+    try {
+      const response = await parseAndSaveParticipants(req.file);
+      return res.json(response);
+    } catch (excp) {
+      return res.status(400).send(`${excp}`);
```

Comment:
Just so we're all aware, we can throw inside the `asyncMiddleware`. The error will be presented as a generic 500 error in `error-handler.js`. If we see a whole bunch of these try/catch blocks, we might want to consider reworking the error handler.

Reply:
OK, but I tried to throw error but it is not showing the appropriate error message.

Reply:
I got it I will fix it different PR.